packages/react-native-gesture-handler/package.json

```diff
@@ -10,14 +10,14 @@
     "ts-check": "yarn tsc --noEmit",
     "format-js": "prettier --write --list-different './src/**/*.{js,jsx,ts,tsx}'",
     "format:js": "yarn format-js",
-    "format:android": "node ./scripts/format-android.js",
+    "format:android": "node ../../scripts/format-android.js",
     "format:apple": "find apple/ -iname *.h -o -iname *.m -o -iname *.cpp -o -iname *.mm | xargs clang-format -i",
     "lint-js": "eslint --ext '.js,.ts,.tsx' src/ && yarn prettier --check './src/**/*.{js,jsx,ts,tsx}'",
     "lint:js": "yarn lint-js",
     "lint:android": "./android/gradlew -p android spotlessCheck -q",
-    "circular-dependency-check": "yarn madge --extensions js,ts,tsx --circular src",
-    "architectures-consistency-check": "node ./scripts/codegen-check-consistency.js",
-    "sync-architectures": "node ./scripts/codegen-sync-archs.js"
+    "circular-dependency:check": "yarn madge --extensions js,ts,tsx --circular src",
+    "architectures:consistency-check": "node ../../scripts/codegen-check-consistency.js",
+    "architectures:sync": "node ../../scripts/codegen-sync-archs.js"
```

Comment:
I'm not sure about the names of the scripts. Those colons don't make much sense to me.

Reply:
Well well well... I agree. But also don't 🤷‍♂️

I think that it is because we are used to the fact that colons were meant to be used as "extension" for the script, while now they represent [global scripts](https://yarnpkg.com/features/workspaces#global-scripts). I've added that because I thought that it will be easier to have those scripts as global ones and be able to run them from any place in the repository. On the other hand, I think we rarely call them by hand, so we can leave them with `-` and use them either by navigating to `packages/-react-native-gesture-handler`, or by using `yarn workspace` command. WDYT?

Reply:
Okay, done in [cd22d2f](https://github.com/software-mansion/react-native-gesture-handler/pull/3506/commits/cd22d2f98b5c87fca5e034a88625aa62460dfc58)